driver-mapping/src/main/java/com/datastax/driver/mapping/PropertyMapper.java

```diff
@@ -40,6 +41,14 @@
  */
 class PropertyMapper {
 
+    private static final Set<Class<? extends Annotation>> COLUMN_ANNOTATIONS = ImmutableSet.of(
```

Comment:
Instead of defining this here, I'd suggest you to reuse `AnnotationParser.VALID_COLUMN_ANNOTATIONS` and `AnnotationParser.VALID_FIELD_ANNOTATIONS` by making them package-private and combining them together.